redirectionio/proxy-symfony/0.1/config/packages/redirectionio.yaml

```diff
@@ -0,0 +1,7 @@
+# redirection.io Configuration
```

Comment:
I don't think this is useful. The file is named `redirectionio.yaml`, and the main configuration key is `redirection_io`.
I would add a link to the documentation instead.